doc/configure/access_log_directives.html

```diff
@@ -205,6 +205,14 @@ <h3><a href="configure/access_log_directives.html#access-log"><code>"access-log"
 <tr><td><code>ssl.server-name</code><td>hostname provided in Server Name Indication (SNI) extension, if any
 </table>
 <table>
+<caption>Upstream Proxy Connection</caption>
+<tr><th>Name<th>Description
+<tr><td><code>proxy.ssl.protocol-version</code><td>SSL protocol version obtained from <a href="https://www.openssl.org/docs/manmaster/ssl/SSL_get_version.html"><code>SSL_get_version</code></a>
+<tr><td><code>proxy.ssl.session-reused</code><td><code>1</code> if the <a href="configure/base_directives.html#ssl-session-resumption">SSL session was reused</a>, or <code>0</code> if not<sup><a href="#note_1" id="#cite_1" title="A single SSL connection may transfer more than one HTTP request.">1</sup></a></sup>
+<tr><td><code>proxy.ssl.cipher</code><td>name of the <a href="https://tools.ietf.org/html/rfc5246#appendix-A.5">cipher suite</a> being used, obtained from <a href="https://www.openssl.org/docs/manmaster/ssl/SSL_CIPHER_get_name.html">SSL_CIPHER_get_name</a>
+<tr><td><code>proxy.ssl.cipher-bits</code><td>strength of the cipher suite in bits
+</table>
+<table>
```

Comment:
Thank you for updating the docs. We often forget doing that, I highly appreciate it.

Please move the changes to `srcdoc/configure/access_log_directives.html`, as that is the source file we use for generating this HTML file. The generation process can be invoked by running `make doc`.